Engines/Wine/Verbs/dotnet452/script.js

```diff
@@ -0,0 +1,48 @@
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Net", "Resource"]);
+include(["Engines", "Wine", "Verbs", "luna"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+include(["Engines", "Wine", "Verbs", "dotnet40"]);
+
+/**
+* Inspired from dotnet40 verb, winetricks dotnet452 and POL4 POL_install_dotnet45
+* -> https://github.com/Winetricks/winetricks/blob/63bc6dbe612d017a0cb6bf6e4cde265162d75bca/src/winetricks#L8523
```

Comment:
I think you should change the line number at the end of the url to fit the part of winetricks that targets dotnet452